kong/core/handler.lua

```diff
@@ -726,8 +725,13 @@ return {
     end
   },
   log = {
-    after = function(ctx)
+    after = function(ctx, mock)
+      if not mock then
```

Comment:
Is it me or is this wrong? Seems like we want to skip this phase when _using_ mock, or else we break a lot of things?